app/assets/stylesheets/application.scss

```
@@ -53,3 +53,13 @@ body {
 .movie-card-body{
   height: 200px;
 }
+
+.text-title {
+  margin-top: 125px;
+}
+
+.card-style {
+  width: 375px;
```

横幅次第で重なっていますので，`max-width` がよいでしょう。
あと，中央に寄せた方がよいですね。

```suggestion
  max-width: 375px;
  margin-left: auto;
  margin-right: auto;
```